src/style.css

```
@@ -1,3 +1,69 @@
+:root {
+  --main-color: #00acdc;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  width: 100%;
+  box-sizing: border-box;
+  padding: 0 50px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.nav__list {
+  display: flex;
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__link {
+  display: inline-block;
+  height: 60px;
+  line-height: 60px;
+  text-decoration: none;
+  color: black;
+  position: relative;
+}
+
+.nav__item {
+  display: flex;
+  text-transform: uppercase;
+  font-weight: 500;
+  margin-right: 20px;
+}
+
+.is-active {
+  position: relative;
+  color: var(--main-color);
+}
+
+.is-active::after {
+  content: '';
+  height: 4px;
+  width: 100%;
+  display: flex;
+  border-radius: 8px;
+  position: absolute;
+  bottom: 0;
+  background-color: var(--main-color);
+}
+
+.nav__link:hover {
+  color: var(--main-color);
+  cursor: pointer;
+}
+
+li:last-child {
```

```suggestion
.nav__item:last-child {
```
use class selector instead of tag selector. Also, move it upper - next to `nav__item` styles